sorc/chgres_cube.fd/input_data.F90

```diff
@@ -6355,6 +6371,15 @@ subroutine handle_grib_error(vname,lev,method,value,varnum, iret,var,var8,var3d)
 
 end subroutine handle_grib_error
 
+!> Read GRIB ...
```

Comment:
If you want to be more descriptive, you can say "Read in soil temperature and soil moisture from a GRIB2 file."

Reply:
Please checkout the branch and update all the doxygen blocks to include at least a sentence or two of description, and document every parameter.